### PR TITLE
virttest.utils_misc.py: add get_guest_cmd_status_output().

### PR DESCRIPTION
**Pull request by CongLi**
Encapsulate session.cmd_status_output() with getting a new session.

Signed-off-by: Cong Li coli@redhat.com

id: 1231041

### CODE REVIEW

**Comment by xutian** [comment]
LGTM

**Comment by CongLi** [comment]
@pingl2016 

Could you help review this one?

Thanks.

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -115,6 +115,26 @@ def join(self, timeout=None, suppress_exception=False):
...
+    Get guest cmd status and output.
+    Encapsulate session.cmd_status_output() with getting a new session.
+
+    :param vm: Guest vm.
+    :param cmd: Cmd will be executed in guest.
+    :param timeout: Timeout for cmd execution in seconds.
+
+    :return: A tuple (status, output) where status is the exit status and
+            output is the output of cmd.
+    """
```
Comment (pingl2016) [inline virttest/utils_misc.py]: shall we need to check whether vm is None before using it?

Comment (CongLi) [inline virttest/utils_misc.py]: @pingl2016 
It's reasonable, could you help review it again?

Thanks.

**Comment by pingl2016** [comment]
It's ok for me.
Acked-by: Ping Li <pingl@redhat.com>

**Comment by xutian** [comment]
merged, thanks @CongLi and @pingl2016

**Pull request merged**